packages/gestalt/src/VideoControls.js

```diff
@@ -154,6 +174,20 @@ class VideoControls extends React.Component<Props> {
             />
           </TapArea>
         </Box>
+        {captionsButton && (
+          <Box padding={2}>
+            <TapArea onTap={this.handleCaptionsChange} fullWidth={false}>
+              <Icon
+                accessibilityLabel="closed-captions"
```

Comment:
@jesselumarie this should be 2 props that are passed in and it should depend on whether we show / hide the captions.

Something along these lines:
```
<Icon accessibilityLabel={captionsButton === 'enabled' ? accessibilityHideCaptionsLabel : accessibilityShowCaptionsLabel} />
```